<launch>
        <node name="cone_height_detector" pkg="rr_iarrc" type="cone_height_detector" output="screen">
                <param name="orange_low_H" type="int" value="5" />
                <param name="orange_high_H" type="int" value="20" />
                <param name="orange_low_S" type="int" value="140" />
                <param name="orange_low_V" type="int" value="140" />

                <!--canny max threshold is set default to 3 * min-->
                <param name="canny_min_threshold" type="int" value="60" />
                <param name="percent_max_distance_transform" type="double" value="0.7" />
                <param name="minimum_area_cut" type="double" value="30" />

                <!--Formula: F = PxHeight * RealDist / RealHeight-->
                <param name="camera_focal_length_y" type="double" value="611.46" />
                <!--9" or .2286m (default meters)-->
                <param name="real_cone_height" type="double" value=".2286" />
                <!--5.5"/2 or .07m (default meters)-->
                <param name="real_cone_radius" type="double" value=".07" />

                <!--The larger the height the lower it will be on the screen-->
                <param name="blockSky_height" type="int" value="500" />
                <param name="blockWheels_height" type="int" value="990" />
                <param name="blockBumper_height" type="int" value="800" />

                <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
        </node>
</launch>
